kythe/go/platform/kzip/kzip.go

```diff
@@ -198,6 +256,30 @@ func ReadConcurrency(n int) ScanOption {
 	return readConcurrency(n)
 }
 
+func (r *Reader) canonicalUnits() (string, []*zip.File) {
+	prefix := r.unitPbPath("") + "/"
+	pos := r.firstIndex(prefix)
+	if pos < 0 {
+		prefix = r.unitPath("") + "/"
+		pos = r.firstIndex(prefix)
+	}
+	res := make([]*zip.File, 0)
```

Comment:
Idiomatically: `var res []*zip.File`

And declare it below the following conditional.